docs/cue-lists/theatre-programming.md

```diff
@@ -23,72 +25,25 @@ Groups also have a User Number.
 
 ## Setting Dimmer/Fixture Levels
 
-````xml
-n <@> <@>
-````
-
-Set fixture **n** to 100%
-
----
-
-````xml
-n <@> v <Enter>
-````
-
-Set fixture **n** to **v%** *(you need to enter 2 digits so "50" for 50%)*
-
----
-
-````xml
-n <Thro> m <@> v <Enter>
-````
-
-Set fixtures **n - m** to **v%**
-
----
-
-````xml
-n <And> m <And> p <@> v <Enter>
-````
-
-Set fixtures **n**, **m**, **p** to **v%**
-
----
-
-````xml
-<Group> n <@> v <Enter>
-````
-
-Set fixtures in group **n** to **v%**
-
----
-
-````xml
-<Clear>
-````
-
-Snap release edited channels back to programmed state -
-*like Cheat on ETC*
-
----
-
-````xml
-t <Clear>
-````
-
-Release edited channels fading over time **t**
+ Keypresses | Operation
+------------|-------------------
+ n \<@\> \<@\> | Set fixture **n** to 100%
```

Comment:
```suggestion
 **n** \<@\> \<@\> | Set fixture **n** to 100%
```
These could be clearer if the **n** in the keypresses section matched the action section